whitelabelco/sofo-foods
# Cp/bug/product pages padding

### Description 💻

Resolves: #161 

### Implementation 💡

fixes spacing issues on individual product pages

refactors individual product pages to use only components for ease of updating

temporarily adds all static data for individual products to `data.tsx`

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click "Products"
- visit each individual product page and confirm spacing + no errors with passed data

### Screenshots/Demos 📷
![Screenshot 2024-10-28 at 3 08 44 PM](https://github.com/user-attachments/assets/44acf24a-b608-48df-86c3-cc39f03ef701)


### Checklist ✅
